src/viewer.ts

```diff
@@ -469,6 +473,7 @@ export class Viewer extends RefCounted implements ViewerState {
   enableAdaptiveDownsampling = new TrackableBoolean(true, true);
   showScaleBar = new TrackableBoolean(true, true);
   showPerspectiveSliceViews = new TrackableBoolean(true, true);
+  hideCrossSectionBackgroundIn3D = new TrackableBoolean(true, true);
```

Comment:
Upon further consideration, I think it is probably unwise to change the default value for existing states.  Instead we should figure out a general mechanism for changing the defaults in new states only, but for now I think it is better to keep it as you had it, and leave the default as false.

Reply:
Yeah that makes a lot of sense, I think this is a good idea (the general mechanism for changing new states). I'll set this back to default false for now though, thanks!